### ISSUE TITLE
Be able to change selected button's tint color

### ISSUE THREAD

**Issue by jaimejazarenoiii** [yonat/RadioGroup#18]
Is it possible to change the outer ring's color if it's selected?

something like this

```
@IBInspectable open dynamic var selectedTintColor: UIColor? {
        didSet {
            layer.borderColor = selectedTintColor.cgColor ?? selectedColor
        }
    }
```
Let me know if it's possible or not, if so, I can make the PR
Thank you.

**Comment by yonat** [comment]
It's not implemented, feel free to make a PR.

**Comment by yonat** [comment]
Implemented in #19
